content/sensu-enterprise/3.2/integrations/graphite.md

```diff
@@ -49,6 +49,8 @@ handler (integration).
 The following attributes are configured within the `{"graphite": {} }`
 [configuration scope][4].
 
+ _PRO TIP: You can override `graphite` attributes on a per-contact basis using [contact routing][5]._
```

Comment:
As I believe this isn't a new feature, should we push this to other versions? Awesome pro-tip for users as they can route the same metric to multiple Graphite instances (prod, backup, etc.).

Reply:
Added this to all versions for these three integrations